requirements.txt

```
@@ -1,5 +1,6 @@
 Babel==2.9.1  # min version = 2.6.0 (Focal with security backports)
 chardet==3.0.4
+cryptography==3.3.2
```

```suggestion
cryptography==2.8
```
When the [Debian](https://packages.debian.org/bullseye/python3-cryptography) and [Ubuntu](https://packages.ubuntu.com/focal/python3-cryptography) versions are different, I prefer to choose the smallest one.